src/tiled/commandmanager.cpp

```diff
@@ -59,17 +59,37 @@ CommandDataModel *CommandManager::commandDataModel()
     return mModel;
 }
 
+void CommandManager::registerMenu(QMenu *menu)
+{
+    mMenus.append(menu);
```

Comment:
I think it would be a good idea to add the actions to this menu here. That way, you don't need to call `populateMenu` from the outside at all and it can be made a private function.